### DISCUSSION TITLE
Project future

### DISCUSSION THREAD

**Discussion by euberdeveloper** [prisma/prisma#5572] [Q&A] [answered]
I am going to use Prisma for an api I have to develop. The api should be working without too frequent maintenance after it is done, how fast does Prisma evolve? I see that great investment and effort gets being given to the project, so will there be in a recent future a version 3? Will the project be kept free?

**Comment by ryands17** [comment, marked as answer]
Hey @euberdeveloper 👋 

Prisma is evolving quite fast as you would be able to see in our [docs](https://www.prisma.io/docs) with many features that have been added and also we have a [roadmap](https://www.notion.so/prismaio/Prisma-Roadmap-50766227b779464ab98899accb98295f) where we explain the features that will be added as development is in progress.

As for a version 3, I don't think we will be shifting to in the near future, but we will be continuously be adding updates/fixes to Prisma 2 and that will be as long as features are added into the roadmap. Prisma's client and Migrate will always be kept free and open source for use.

I hope that answers your question :)

**Reply by euberdeveloper** [reply]
Yes, thank you!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
